int32 object_ID
float32 x
float32 y
float32 velx
float32 vely
